Doc/library/typing.rst

```diff
@@ -991,7 +991,9 @@ These can be used as types in annotations using ``[]``, each having a unique syn
 
       class Starship:
           stats: ClassVar[dict[str, int]] = {} # class variable
-          damage: int = 10                     # instance variable
+
+          def __init__(self, damage: int):
+              self.damage = damage           # instance variable
```

Comment:
We should still keep the instance variable type annotation at the class-level:
```
      class Starship:
          stats: ClassVar[dict[str, int]] = {} # class variable
          damage: int = 10                     # instance variable
          
          def __init__(self, damage: int): ...
```
This is important because the confusion that ClassVar resolves is what the intention of a class-level annotations is